### PR TITLE
added echo answer to read_answer_with_default()

### PR DESCRIPTION
**Pull request by BenjaminJPerry**
added echo following `read answer` to echo User response or default parameters.

### CODE REVIEW

File: _run_gbs_qc [hunk 1]
```diff
@@ -17,11 +17,14 @@ function send_mail() {
...
    if [ $INTERACTIVE == yes ]; then
       read answer
+      echo "User command: $answer"
       if [ -z "$answer" ]; then
          answer=$@
+         echo "Default command: $answer"
       fi
    else
       answer=$@
+      echo "Default command: $answer"
```
Comment (BenjaminJPerry) [inline _run_gbs_qc:27]: Should actually delete this line as it is used in the non-interactive mode

**Review by afmcc — COMMENTED** [review]
thanks looks good - just wondering if it might be clearer phrasing to say "User response:" and "Default response:" ? (rather than "User command:" and "Default command:")

**Comment by BenjaminJPerry** [comment]
Sounds good to me

**Comment by BenjaminJPerry** [comment]
Clarification to the CLI feedback

**Pull request merged**